releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,25 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.2.0 - January 13, 2020
+
+* A number of configuration parameters have renamed/replaced/removed. See [starter](https://github.com/microsoft/spring-cloud-azure/blob/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md) for more info.
+
+```properties
+spring.cloud.azure.appconfiguration.watch.delay -> spring.cloud.azure.appconfiguration.cache.expiration
+
+# Endpoint Requires URI value
+spring.cloud.azure.appconfiguration.stores[0].name -> spring.cloud.azure.appconfiguration.stores[0].endpoint
+```
+
+* AzureCloudConfigWatch is now named AzureCloudConfigRefresh. Also AzureCloudConfigRefresh is always created and can be access via dependency injection to allow control over refreshes outside of ServletRequestHandledEvents.
+* System Assigned Credential no longer needs client id set.
```

Comment:
* Fixed the bug that system-assigned managed identity no longer needs client id to be set.